backend/test/resources.js

```diff
@@ -49,6 +59,7 @@ describe("GET /resources", () => {
       .get(`/api/resources`)
       .expect(200);
     expect(res.body.result).to.be.an("array").that.is.empty;
+    expect(didCheckIsAdmin()).to.be.true;
```

Comment:
This, and a few of the ones below result in failed tests because it will assert that it checked the user was admin yet GET /resources only checks if the user is a volunteer (as it should);